lib/pronto/rubocop/offense_line.rb

```diff
@@ -95,7 +95,11 @@ def autocorrect_team
       end
 
       def cop_class
-        patch_cop.registry.find_by_cop_name(offense.cop_name)
+        if patch_cop.registry.respond_to?(:find_by_cop_name)
```

Comment:
`find_by_cop_name` has been available since version 0.58.0 (rubocop-hq/rubocop@0ad211b684834bb65aaf053f7bdebca2b94ee9f0)

Should we change `pronto-rubocop.gemspec` ?

Reply:
As long as `0.58.0` is compatible with Ruby 2.3, I don't mind bumping up the minimum required version to that.